webpack/scenes/Subscriptions/Manifest/DeleteManifestModalText.js

```diff
@@ -0,0 +1,22 @@
+const question = __('Are you sure you want to delete the manifest?');
+const note = __(`Note: Deleting a subscription manifest is STRONGLY discouraged.
+                 Deleting a manifest will:`);
+const l1 = __('Delete all subscriptions that are attached to running hosts.');
+const l2 = __('Delete all subscriptions attached to activation keys.');
+const l3 = __('Disable Red Hat Insights.');
+const l4 = __(`Require you to upload the subscription-manifest and re-attach
+               subscriptions to hosts and activation keys.`);
+const debug = __(`This action should only be taken in extreme circumstances or
+                  for debugging purposes.`);
+
+const DeleteManifestModalText = `<p>${question}</p>
+<p>${note}</p>
+<ul class="list-aligned">
+  <li>${l1}</li>
```

Comment:
It looks like this is actually being evaluated as html so that won't work. When I tried `<p>{__('does this work?')}</p>`, it looks like this:
![here](http://www.clipular.com/c/5214192471375872.png?k=2r2mTzeocTUJtFiZgpbAE-9QtzI)

It makes sense since we are passing to [dangerouslySetInnerHTML](https://github.com/Katello/katello/pull/7365/files#diff-249974991321c6e620eba5f5a0a65abdR212)

Reply:
> It makes sense since we are passing to dangerouslySetInnerHTML

Oh, right.  And same thing with this?

```jsx
<p>${__('does this work?'}</p>
```

Reply:
Hmm, I wonder if we should update the `Dialog` component to accept children too. That would probably solve the problem because we could easily pass array of components instead of a string.